.github/workflows/release.yml

```diff
@@ -13,16 +13,6 @@ jobs:
     strategy:
       matrix:
         config:
-          - {
-              os: "ubuntu-18.04",
```

Comment:
This is our main linux release. Bump to 20.04 instead?

Reply:
Oof, thanks for the catch.  For some reason I thought we released off latest.  Done!